src/regtech_user_fi_management/config.py

```diff
@@ -28,8 +29,8 @@ class Settings(BaseSettings):
     admin_scopes: Set[str] = set(["query-groups", "manage-users"])
     db_logging: bool = True
 
-    def __init__(self, **data):
-        super().__init__(**data)
+    def __init__(self, _env_file: DotenvType | None = ENV_FILE_SENTINEL, **data):
+        super().__init__(_env_file=_env_file, **data)
```

Comment:
Realized we don't need this.  Because the Settings was already taking in **data, code using Settings can already pass in Settings(_env_file='something') and that will be passed to BaseSettings.  Same would go with the KeycloakSettings in commons-api.